do_cmake.sh

```diff
@@ -8,6 +8,12 @@ fi
 
 PYBUILD="2"
 source /etc/os-release
+
+if [ "$(uname)" == FreeBSD ] ; then
+  DWITH_RADOSGW_AMQP_ENDPOINT="OFF"
+  DWITH_RADOSGW_KAFKA_ENDPOINT="OFF"
```

Comment:
this change does not look right. probably you could backport 46b3d671b4408a766aaed379274242ad4b38afb3 as well?